core/psh/Makefile

```diff
@@ -4,7 +4,11 @@
 # Copyright 2018, 2019, 2020, 2021 Phoenix Systems
 #
 
-OBJS = $(filter-out psh psh-empties help, $(subst .c,,$(subst ./core/psh/,, $(shell find ./core/psh -name '*.c'))))
+ifdef DEFUSR_PASSWD
+CFLAGS += -DDEFUSR_PASSWD=\"$(DEFUSR_PASSWD)\"
+endif
+
+OBJS = bind cat exec kill ls mem mkdir mount nc perf ping ps reboot runfile sync sysexec top touch
```

Comment:
this for sure are not objects, so the name `OBJS` in invalid here

Reply:
Changed variables names. Now OBJS in fact holds list of object files